src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt

```diff
@@ -116,21 +117,31 @@ class XmppApi(
                             mucClient.sendStanza(JibriPresenceHelper.createPresence(status, it))
                         }
                     }
+
+                    val updatePresenceStanza: (Presence) -> Unit = { presence ->
```

Comment:
i think my other comment got lost.  can you move this to a private member function?